lib/offsite_payments/notification.rb

```diff
@@ -30,11 +30,10 @@ def gross_cents
       (gross.to_f * 100.0).round
     end
 
-    # This combines the gross and currency and returns a proper Money object.
-    # this requires the money library located at http://rubymoney.github.io/money/
     def amount
-      return Money.new(gross_cents, currency) rescue ArgumentError
-      return Money.new(gross_cents) # maybe you have an own money object which doesn't take a currency?
+      amount = gross ? gross.to_d : 0
```

Comment:
because `nil.to_d` will raise instead of returning 0. We want to keeo the old behaviour
```
Money.new((gross.to_f * 100.0).round, currency)
```
here `.to_f` would return `0` when gross is `nil`.